app/controllers/Api2.scala

```diff
@@ -16,6 +16,9 @@ import util.atom.MediaAtomImplicits
 import play.api.libs.json._
 import play.api.mvc._
 
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Should we use the Play execution context instead? 

https://www.playframework.com/documentation/2.5.x/ThreadPools#Using-the-default-thread-pool